swagger/sales/coupons.json

```diff
@@ -0,0 +1,96 @@
+{
+  "host": "api.vchost.co",
+  "swagger": "2.0",
+  "info": {
+    "title": "API V3",
```

Comment:
I assume that any sales related endpoint will be added to this json. Due to limitation of the developer portal, the json needs to be a combined json of all of them. so:
Please add the name of the API name, in this case its "Sales" you need the title "Sales" and a short description, like this:
"info": {
        "title": "Sales",
        "description": "A set of endpoints to manage sales activities between businesses and clients, including, payments, coupons,",
        "version": "1.0",
        "contact": {}
    },